frontend/src/app/app.component.ts

```diff
@@ -1,9 +1,21 @@
-import { Component } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Component, Inject } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
 })
 export class AppComponent {
-  title = 'HTTAT';
+  isDarkMode = false;
+
+  constructor(@Inject(DOCUMENT) private document: Document) {}
```

Comment:
Can this be moved into the header? You might have to use `this.document` to query for the `body` element.

Reply:
also done.